src/com/google/firebase/AuthenticatedDAO.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * @license
+ * Copyright 2018 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
+  package: 'com.google.firebase',
+  name: 'AuthenticatedDAO',
```

Comment:
This should probably be named something like `AwaitAuthenticationDAO`; it's not an `AuthenticatedDAO` in the server sense, which is misleading.

Reply:
Done.